### PR TITLE
Project Hello World

### PR DESCRIPTION
**Pull request by ppmarq1**
Project Hello World with pull request to main

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,4 @@
-# Hello-World
\ No newline at end of file
+# Hello-World
+#Project built with HTML and CSS
+
+#It is included linters and gitignore
```
Comment (omar-muhamad) [inline README.md]: - [ ] Please make your README file more descriptive by using [this template](https://github.com/microverseinc/readme-template) and at least edit these sections (Project name, Project description, Getting Started, Built with, Author, and License) with the proper info and please delete any other unused sections.

**Review by omar-muhamad — CHANGES_REQUESTED** [review]
# Changes Required ♻️
### Hi @ppmarq1,

Nice job, you are almost there.



## (Highlights) Good Points: :+1:
- You documented your work professionally. :heavy_check_mark:
- Github flow was correctly used. :heavy_check_mark:

## (Changes Required) Aspects to improve: ♻️

- [ ] Please fix the `Stylelint` error and make sure that all linters checks are green.

- [ ] Nice work, your commit messages are meaningful, Next time please make sure that you capitalize the first letter in your commit message.

- [ ] Next time kindly make your branch name more descriptive about what you implement in that branch for example in this project you can call it `hello-microverse`.

- [ ] Please check the inline comments below.

## [Optional] suggestions:

- [ ] [Optional] In the Pull Request description it would be better if you make checklist or bullet points with all changes you did in that Pull Request.

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏**

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: README.md [hunk 2]
```diff
@@ -1,4 +1,16 @@
...
+Built With
+HTML
+CSS
+
+
+Authors
+👤 Pedro Marques
+
+GitHub: @ppmarq1
+Twitter: @ppmarq1
```
Comment (ShadyShawkat) [inline README.md:16]: - [ ] [OPTIONAL] Please be sure to use proper styling for the `README.md`, you can check the raw text of Microverse's readme template to do so.

**Review by ShadyShawkat — APPROVED** [review]
## Hi @ppmarq1, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Image

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**